cloud-functions/utils.py

```diff
@@ -0,0 +1,57 @@
+import logging
+import flask
+
+""" A Class to hold commonly used functions
+that can be reused by the different cloud functions
+
+TODO: We may want to move this into the SDK
```

Comment:
I think you're right here - ideally the cloud functions code is as minimal as possible, pulling any shared code from the SDK. Plus the way that we will bundle these functions w/ terraform means that they won't be able to reference the directory above themselves, so it probably makes the most sense to slide this over to the other repo.

Reply:
Yes I think having this moved over to the SDK project makes the most sense and will clean up the GCP Cloud Function code a great deal.  Maybe we need to open a new ticket for us to handle this.